src/cascadia/TerminalApp/ITab.idl

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "Command.idl";
+
+namespace TerminalApp
+{
+    interface ITab
+    {
+        String Title { get; };
+        Windows.UI.Xaml.Controls.IconSource IconSource { get; };
+        Command SwitchToTabCommand { get; };
+
+        Microsoft.UI.Xaml.Controls.TabViewItem GetTabViewItem();
+        Windows.UI.Xaml.UIElement GetTabContent();
+        String GetActiveTitle();
+
+        Boolean IsFocused();
+        void SetFocused(Boolean focused);

```

Comment:
may wanna make this a property! or, copy the `Focus`/`TryFocus` method from xaml UIElement/Control (!) and have the same signature